packages/medusa/src/api/admin/products/middlewares.ts

```diff
@@ -102,7 +97,7 @@ export const adminProductRoutesMiddlewares: MiddlewareRoute[] = [
   {
     method: ["POST"],
     matcher: "/admin/products/import",
-    middlewares: [upload.single("file")],
```

Comment:
Should this PR also change the import products UI? Otherwise I guess it will be broken until we do it

Reply:
The imports UI works fine. However, I have updated the `js-sdk` to now make 3 calls instead of one, when a new file is uploaded.